src/graphql/graphql.tsx

```diff
@@ -0,0 +1,693 @@
+
+export type Market = {
+  __typename?: 'PmmStorageMarketMapOrdered';
+  id: any;
+  metadataIpfsHash: string;
+  metadataDescription: string;
+  metadataAdjudicator: string;
+  marketsState: string;
+  marketId: any;
+  txContext: {
+    __typename?: 'PublicTxContext';
+    operationGroupNumber: number;
+    operationNumber: number;
+    contentNumber: number;
+    blockInfo: { __typename?: 'PublicLevel'; bakedAt: any; block: number };
+  };
+  marketInfo: {
+    __typename?: 'PmmStorageMarketMap';
+    marketBootstrapped: Array<{
+      __typename?: 'PmmStorageMarketMapMarketbootstrapped';
+      resolutionResolvedAtBlock: any;
+      marketBootstrappedBootstrappedAtBlock: any;
+      auctionRewardCurrencyPool: any;
+      liquidityRewardPool: any;
+      marketCurrencyPool: any;
+      bootstrapYesProbability: any;
+      winningPrediction?: string;
+    }>;
+    auctionRunning: Array<{
+      __typename?: 'PmmStorageMarketMapAuctionrunning';
+      auctionRunningAuctionPeriodEnd: any;
+      auctionRunningQuantity: any;
+      auctionRunningYesPreference: any;
+      auctionRunningUniswapContribution: any;
+    }>;
+  };
+};
+
+export type MarketSubscription = {
+  __typename?: 'Subscription';
+  markets: Array<Market>;
+};
+
+export type MarketBetsSubscriptionVariables = Exact<{
+  marketId: Maybe<Scalars['BigFloat']>;
+}>;
+
+export type MarketBetsSubscription = {
+  __typename?: 'Subscription';
+  allPmmStorageLiquidityProviderMapLivesList: Array<{
+    __typename?: 'PmmStorageLiquidityProviderMapLive';
+    id: any;
+    marketId: any | null;
+    originator: string | null;
+    betDetails: {
+      __typename?: 'PmmStorageLiquidityProviderMap';
+      bets: Array<{
+        __typename?: 'PmmStorageLiquidityProviderMapBet';
+        probability: any | null;
+        quantity: any | null;
+      }>;
+    } | null;
+    txContext: {
+      __typename?: 'PublicTxContext';
+      operationGroupNumber: number;
+      operationNumber: number;
+      contentNumber: number;
+      txHash: string;
+      blockInfo: { __typename?: 'PublicLevel'; bakedAt: any | null; block: number } | null;
+    };
+  }>;
+};
+
+export type TokenSupplyByTokenIdSubscriptionVariables = Exact<{
+  tokens: Maybe<Array<Scalars['BigFloat']> | Scalars['BigFloat']>;
+}>;
+
+export type TokenSupplyByTokenIdSubscription = {
+  __typename?: 'Subscription';
+  supplyMaps: Array<{
+    __typename?: 'PmmStorageSupplyMapLive';
+    id: any;
+    tokenId: any | null;
+    totalSupply: any | null;
+    tokenReserve: any | null;
+    txContext: {
+      __typename?: 'PublicTxContext';
+      operationGroupNumber: number;
+      operationNumber: number;
+      contentNumber: number;
+      txHash: string;
+      blockInfo: { __typename?: 'PublicLevel'; bakedAt: any | null; block: number } | null;
+    };
+  }>;
+};
+
+export type UserBetsSubscriptionVariables = Exact<{
+  originator: Maybe<Scalars['String']>;
+}>;
+
+export type UserBetsSubscription = {
```

Comment:
i think the typings and the subscriptions should be split into different files

Reply:
maybe in a different PR.